account_payment_fee_transfer/models/account_payment.py

```diff
@@ -0,0 +1,127 @@
+        string="Deduct From",
+        domain="[('id', 'in', [journal_id, destination_journal_id])]",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+
+    def _prepare_payment_moves(self):
+        """ Update fee transfer and balance by creating a list of python dictionary
+
+        Example 1: Transfer, BANK1 -> BANK2 (fee on BANK1):
+
+        Account             | Debit     | Credit
+        =========================================================
+        Journal Entry 1
+        ---------------
+        BANK1               |           |   1000.0
+        BANK1               |           |     10.0        --> Fee
+        ACCOUNT FEE         |     10.0  |
+        TRANSFER            |   1000.0  |
+
+        Journal Entry 2
+        ---------------
+        TRANSFER            |           |   1000.0
+        BANK2               |   1000.0  |
+
+        Example 2: Transfer, BANK1 -> BANK2 (fee on BANK2):
+
+        Account             | Debit     | Credit
+        =========================================================
+        Journal Entry 1
+        ---------------
+        BANK1               |           |   1000.0
+        TRANSFER            |   1000.0  |
+
+        Journal Entry 2
+        ---------------
+        TRANSFER            |           |   1000.0
+        BANK2               |   1000.0  |
+        ACCOUNT FEE         |     10.0  |
+        BANK2               |           |     10.0        --> Fee
+
+        :return: A list of Python dictionary to be passed to env['account.move'].create.
+        """
+        res = super(AccountPayment, self)._prepare_payment_moves()
+        for payment in self:
+            company_currency = payment.company_id.currency_id
+            if payment.payment_type != "transfer" or payment.fee_transfer == 0.0:
+                continue
+            for move in res:
+                journal_id = move.get("journal_id", False)
+                lines = move.get("line_ids")
+                # fee_transfer_name = _('Fee transfer from %s') \
+                #     % payment.deduct_journal_id.name
+
+                # Manage currency.
+                if payment.currency_id == company_currency:
+                    # Single-currency.
+                    balance = payment.fee_transfer
+                    currency_id = False
+                else:
+                    # Multi-currencies.
+                    balance = payment.currency_id._convert(
+                        payment.fee_transfer,
+                        company_currency,
+                        payment.company_id,
+                        payment.payment_date,
+                    )
+                    currency_id = payment.currency_id.id
+
+                # Add fee transfer in journal deduct.
+                if journal_id == payment.deduct_journal_id.id:
+                    credit_account = (
+                        payment.deduct_journal_id.default_debit_account_id.id
```

Comment:
As we get for credit account. A little more inline.
```suggestion
                        payment.deduct_journal_id.default_credit_account_id.id
```